src/main/java/io/vinyldns/java/model/batch/AddChangeInput.java

```diff
@@ -17,38 +17,16 @@
 import io.vinyldns.java.model.record.data.RecordData;
 import java.util.Objects;
 
-public class AddChangeInput implements ChangeInput {
-  private String inputName;
-
-  private RecordType type;
-
+public class AddChangeInput extends ChangeInput {
   private Long ttl;
```

Comment:
We should convert these to `final` since there's only one constructor and no need for setters.